e2e-tests/commands/insurance/complete-export-value-form.js

```diff
@@ -0,0 +1,33 @@
+import { FIELD_VALUES } from '../../constants';
+import { INSURANCE_FIELD_IDS } from '../../constants/field-ids/insurance';
+import { multipleContractPolicyExportValuePage } from '../../pages/insurance/policy';
+import { field } from '../../pages/shared';
+import application from '../../fixtures/application';
+
+const {
+  POLICY: {
+    EXPORT_VALUE: {
+      MULTIPLE: {
+        TOTAL_SALES_TO_BUYER,
+        MAXIMUM_BUYER_WILL_OWE,
+      },
+    },
+  },
+} = INSURANCE_FIELD_IDS;
+
+/**
+ * completeExportValueForm
+ * Complete the "Export value" form
+ * @param {Object} Object with flags completing and submitting export value form
+ * - policyType: type of policy
+ * - policyMaximumValue: should submit an application with the maximum value of 500000
+ */
+const completeExportValueForm = ({ policyType = FIELD_VALUES.POLICY_TYPE.SINGLE }) => {
```

Comment:
not a blocker, we can dereference SINGLE and MULTIPLE

Reply:
As discussed will address in the next PR